<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="1" total="2" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-10-31T15:08:07 IST" name="Default suite" finished-at="2024-10-31T15:08:26 IST" duration-ms="19028">
    <groups>
    </groups>
    <test started-at="2024-10-31T15:08:07 IST" name="Default test" finished-at="2024-10-31T15:08:26 IST" duration-ms="19028">
      <class name="testSuite.AddEmployeePageTests">
        <test-method is-config="true" signature="getCurrentMethodAndClass(org.testng.ITestResult)[pri:0, instance:testSuite.AddEmployeePageTests@8f4ea7c]" started-at="2024-10-31T15:08:07 IST" name="getCurrentMethodAndClass" finished-at="2024-10-31T15:08:07 IST" duration-ms="6" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name={null} status=CREATED method=AddEmployeePageTests.testCase01()[pri:0, instance:testSuite.AddEmployeePageTests@8f4ea7c] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getCurrentMethodAndClass -->
        <test-method is-config="true" signature="getTestData()[pri:0, instance:testSuite.AddEmployeePageTests@8f4ea7c]" started-at="2024-10-31T15:08:07 IST" name="getTestData" finished-at="2024-10-31T15:08:07 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getTestData -->
        <test-method is-config="true" signature="loadData()[pri:0, instance:testSuite.AddEmployeePageTests@8f4ea7c]" started-at="2024-10-31T15:08:07 IST" name="loadData" finished-at="2024-10-31T15:08:13 IST" duration-ms="5783" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loadData -->
        <test-method signature="testCase01()[pri:0, instance:testSuite.AddEmployeePageTests@8f4ea7c]" started-at="2024-10-31T15:08:13 IST" name="testCase01" finished-at="2024-10-31T15:08:26 IST" duration-ms="13209" status="FAIL">
          <exception class="java.lang.reflect.InvocationTargetException">
            <full-stacktrace>
              <![CDATA[java.lang.reflect.InvocationTargetException
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at utils.FrameworkUtils.performMethod(FrameworkUtils.java:97)
at testSuite.AddEmployeePageTests.testCase01(AddEmployeePageTests.java:21)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
Caused by: java.lang.AssertionError: Element is not displayed and not enabled expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at utils.ElementUtils.textField(ElementUtils.java:85)
at pages.AddEmployeePage.uploadPhoto(AddEmployeePage.java:30)
at pages.AddEmployeePage.performSave(AddEmployeePage.java:57)
... 34 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.reflect.InvocationTargetException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCase01 -->
      </class> <!-- testSuite.AddEmployeePageTests -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
